examples/light-switch-app/light-switch-common/light-switch-app.matter

```diff
@@ -2563,6 +2589,15 @@ endpoint 0 {
     ram      attribute clusterRevision default = 1;
   }
 
+  server cluster UnitLocalization {
```

Comment:
Why is this here?  It shouldn't need to be here.  If ZAP says it does, ZAP is wrong.

Reply:
`<include cluster="Unit Localization" client="false" server="true" clientLocked="true" serverLocked="false"></include>`
By default the XML says that this cluster is true with `server="true"` .That is why ZAP is warning the user. Should this be turned to false by default?

Reply:
Based on the user, this can go either way with whether the user needs Unit localization or not because by default it is expected to be enabled.